app/services/mailer.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const nodemailer = require('nodemailer');
+const config = require('../../config');
+
+let transporter = null;
+
+const getTransporter = () => {
+  if (!transporter) {
+    transporter = nodemailer.createTransport({
+      host: config.common.mailer.mailerHost,
+      port: config.common.mailer.mailerPort,
+      secure: false,
+      auth: {
+        user: config.common.mailer.mailerUser,
+        pass: config.common.mailer.mailerPass
+      }
+    });
+  }
+  return transporter;
+};
```

Comment:
You could define this as a constant directly, what advantages fives this to you?

Reply:
If I used const, It could not be mocked for some reason